package.json

```diff
@@ -4,35 +4,37 @@
   "description": "A JavaScript library for rendering music notation and guitar tablature.",
   "exports": {
     ".": {
-      "types": "./build/src/entry/vexflow.d.ts",
-      "require": "./build/vexflow.js",
-      "import": "./build/esm/vexflow.js"
+      "types": "./build/types/entry/vexflow.d.ts",
+      "require": "./build/cjs/vexflow.js",
+      "import": "./build/esm/entry/vexflow.js"
     },
     "./core": {
-      "types": "./build/src/entry/vexflow-core.d.ts",
-      "require": "./build/vexflow-core.js",
-      "import": "./build/esm/vexflow-core.js"
+      "types": "./build/types/entry/vexflow-core.d.ts",
+      "require": "./build/cjs/vexflow-core.js",
+      "import": "./build/esm/entry/vexflow-core.js"
     },
     "./core-with-bravura": {
-      "types": "./build/src/entry/vexflow-core-with-bravura.d.ts",
-      "require": "./build/vexflow-core-with-bravura.js",
-      "import": "./build/esm/vexflow-core-with-bravura.js"
+      "types": "./build/types/entry/vexflow-core-with-bravura.d.ts",
+      "require": "./build/cjs/vexflow-core-with-bravura.js",
+      "import": "./build/esm/entry/vexflow-core-with-bravura.js"
     },
     "./core-with-gonville": {
-      "types": "./build/src/entry/vexflow-core-with-gonville.d.ts",
-      "require": "./build/vexflow-core-with-gonville.js",
-      "import": "./build/esm/vexflow-core-with-gonville.js"
+      "types": "./build/types/entry/vexflow-core-with-gonville.d.ts",
+      "require": "./build/cjs/vexflow-core-with-gonville.js",
+      "import": "./build/esm/entry/vexflow-core-with-gonville.js"
     },
     "./core-with-petaluma": {
-      "types": "./build/src/entry/vexflow-core-with-petaluma.d.ts",
-      "require": "./build/vexflow-core-with-petaluma.js",
-      "import": "./build/esm/vexflow-core-with-petaluma.js"
+      "types": "./build/types/entry/vexflow-core-with-petaluma.d.ts",
+      "require": "./build/cjs/vexflow-core-with-petaluma.js",
+      "import": "./build/esm/entry/vexflow-core-with-petaluma.js"
```

Comment:
These entries describe the various import / require paths that our npm package offers.

```
"."              --->  import { Vex } from "vexflow"
"./core"         --->  import { Vex } from "vexflow/core" 
... etc
```